README.md

```diff
@@ -144,5 +144,5 @@ constructorIO.trackPurchase(customerIDs: ["123-AB", "456-CD"])
 constructorIO.trackBrowseResultsLoaded(filterName: "Category", filterValue: "Snacks", resultCount: 674)
 
 // Track when a browse result is clicked
-constructorIO.trackBrowseResultClick(filterName: "Category", filterValue: "Snacks", customerID: "7654321-BA", resultPositionOnPage: 4)
+constructorIO.trackBrowseResultClick(filterName: "Category", filterValue: "Snacks", customerID: "7654321-BA", resultPositionOnPage: 4, resultID: "179b8a0e-3799-4a31-be87-127b06871de2")
```

Comment:
there is a sectionName argument before resultID, so we should add the sectionName as well so something like `sectionName: "Products", resultID: "179b8a0e-3799-4a31-be87-127b06871de2"`

Reply:
e8ab7e4